recipes/vtk-m/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "vtk-m" %}
+{% set version_major = "2" %}
+{% set version_minor = "0" %}
+{% set version_patch = "0" %}
+{% set version = version_major + "." + version_minor + "." + version_patch %}
+{% set common_flags = "-DCMAKE_BUILD_TYPE=Release -DVTKm_ENABLE_EXAMPLES=ON -DVTKm_INSTALL_EXAMPLES=OFF" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.kitware.com/vtk/vtk-m/-/archive/v{{ version }}/{{ name }}-v{{ version }}.tar.gz
+  sha256: 32643cf3564fa77f8e2a2a5456a574b6b2355bb68918eb62ccde493993ade1a3
+
+build:
+  script:
+    - cmake -GNinja -S $SRC_DIR  -B build -DCMAKE_INSTALL_PREFIX=$PREFIX {{ common_flags }}  # [unix]
+    - cmake -GNinja -S %SRC_DIR% -B build -DCMAKE_INSTALL_PREFIX=%PREFIX% {{ common_flags }}  # [win]
+    - cmake --build build -j
+    - ./build/examples/smoke_test/smoke_test  # [unix]
+    - build\examples\smoke_test\smoke_test.exe  # [win]
+    - cmake --install build
+  number: 0
```

```suggestion
build:
  script:
    - cmake -GNinja -S $SRC_DIR  -B build -DCMAKE_INSTALL_PREFIX=$PREFIX {{ common_flags }}  # [unix]
    - cmake -GNinja -S %SRC_DIR% -B build -DCMAKE_INSTALL_PREFIX=%PREFIX% {{ common_flags }}  # [win]
    - cmake --build build -j
    - ./build/examples/smoke_test/smoke_test  # [unix]
    - build\examples\smoke_test\smoke_test.exe  # [win]
    - cmake --install build
  number: 0
  run_exports:
    - {{ pin_subpackage( name|lower, max_pin='x.x') }}
```
Packages with headers and shared objects should have run exports. Please adjust the maximum pin according to your library's ABI compatability guarantees.